auth-web/tests/unit/components/TransactionsDataTable.spec.ts

```diff
@@ -97,9 +98,10 @@ describe('TransactionsDataTable tests', () => {
     expect(row1Cells.at(3).text()).toContain('January 24, 20233:09 PM')
```

Comment:
can you alter the test setup data to not include the timezone in the date string? That way the test will secure this bug

Reply:
sounds good